wandb/sdk/service/server.py

```diff
@@ -0,0 +1,127 @@
+"""wandb server.
+
+Start up grpc or socket transport servers.
+"""
+
+from concurrent import futures
+import logging
+import sys
+from typing import Optional
+
+import grpc
+import wandb
+from wandb.proto import wandb_server_pb2_grpc as spb_grpc
+
+from . import port_file
+from .server_sock import SocketServer
+from .streams import StreamMux
+
+
+class WandbServer:
+    _pid: Optional[int]
+    _grpc_port: Optional[int]
+    _sock_port: Optional[int]
+    _debug: bool
+    _serve_grpc: bool
+    _serve_sock: bool
+    _sock_server: Optional[SocketServer]
+
+    def __init__(
+        self,
+        grpc_port: int = None,
+        sock_port: int = None,
+        port_fname: str = None,
+        address: str = None,
+        pid: int = None,
+        debug: bool = False,
+        serve_grpc: bool = False,
+        serve_sock: bool = False,
+    ) -> None:
+        self._grpc_port = grpc_port
+        self._sock_port = sock_port
+        self._port_fname = port_fname
+        self._address = address
+        self._pid = pid
+        self._debug = debug
+        self._serve_grpc = serve_grpc
+        self._serve_sock = serve_sock
+        self._sock_server = None
+
+        if grpc_port:
+            _ = wandb.util.get_module(  # type: ignore
+                "grpc",
+                required="grpc port requires the grpcio library, run pip install wandb[grpc]",
+            )
+
+        debug = True
+        if debug:
```

Comment:
This condition is always going ot be `True`, do you want to remove line 56?

Reply:
changed the default in the function to debug=True instead.